include/ocpp/v201/charge_point.hpp

```diff
@@ -54,6 +54,12 @@ class ChargePoint : ocpp::ChargingStationBase {
 
     std::map<int32_t, ChangeAvailabilityRequest> scheduled_change_availability_requests;
 
+    std::map<std::string,
+             std::map<std::string, std::function<DataTransferResponse(const std::optional<std::string>& msg)>>>
+        data_transfer_callbacks;
+    std::map<std::string, std::function<void(Call<DataTransferRequest> call)>> data_transfer_pnc_callbacks;
```

Comment:
We dont need the data_transfer_pnc_callbacks for 2.0.1 because this mechanism of wrapping the PnC messages into the DataTransfer.req message is only necessary for supporting PnC in 1.6. This functionality is supported by other messages in 2.0.1